[master < Export-csv[ Write docs for csv graph export (#1025)
memgraph/docs: mage/query-modules/python/export-util.md

| `separateHeader` | `False` | Indicates whether the CSV file header should be separated into its own file. If `True`, a file named `header.csv` will be created in the specified or default directory.|
| `stream` | `False` | Indicates whether the data should be returned as a stream rather than a file. If `True`, no file will be saved. |

#### Output:

- `data: string` ➡ if stream is `True`, a string stream representation of data, otherwise `""`.
- `path: string` ➡ path to the exported CSV file.

#### File structure:

The CSV file columns are structured this way:

| _id | _labels| node_properties_sorted | _start | _end | _type | relationship_properties_sorted |
|------|-----|--------|-------------|--------|------|-------|
|node id| node labels | alphabetically sorted properties of all nodes (all node properties present in the graph) | id of the start node of a relationship | id of the end node of a relationship | type of relationship | alphabetically sorted properties of all relationships (all relationship properties present in the graph) |

> For example, consider a simple graph, created with this cypher query:

Reviewer: ```suggestion
For example, consider a simple graph created with the following Cypher query:
```

```cypher
CREATE (d:Dog {name: "Rex", breed: "Dalmatian"})-[i:IS_OWNED_BY {rel_property: 30}]->(h:Human {name: "Carl", age: 50});
```

> The id of the Dog node is 0, and the id of the Human node is 1.
> Exporting would result in a CSV file structured this way:

Reviewer: ```suggestion
The `Dog` node's ID is 0, and the `Human` node's ID is 1.
Exporting would result in a CSV file structured in the following way:
```

```

</TabItem>

<TabItem value="result1">

| _id | _labels | age | branch | breed | name | years_of_service | _start | _end | _type | duration | rel_property |
|-----|-----------------|-----|--------|----------|------|------------------|--------|------|-----------------|----------------------------|-----------------|
| 0 | :Dog | | | Dalmatian| Rex | | | | | | |
| 1 | :Human | 50 | | | Carl | | | | | | |
| 2 | :Human:Soldier | | Army | | | | | | | | |
| 3 | :Dog:K9 | | | | Bolt | 3 | | | | | |
| | | | | | | | 0 | 1 | IS_OWNED_BY | | 30 |
| | | | | | | | 2 | 3 | TRAINS | duration(10 days,0:00:00) | |

</TabItem>

<TabItem value="result2">

```plaintext

"_id","_labels","age","branch","breed","name","years_of_service","_start","_end","_type","duration","rel_property"
"0",":Dog","","","Dalmatian","Rex","","","","","",""
"1",":Human","50","","","Carl","","","","","",""
"2",":Human:Soldier","","Army","","","","","","","",""
"3",":Dog:K9","","","","Bolt","3","","","","",""
"","","","","","","","0","1","IS_OWNED_BY","","30"
"","","","","","","","2","3","TRAINS","duration(10 days, 0:00:00)",""

```

</TabItem>

</Tabs>

#### Usage with configurations:

> In this section, usage with configurations is displayed. The graph used is the same one as in usage without configurations.

Reviewer: ```suggestion
In this section, usage with configurations is displayed. The same graph is used as in the usage without configurations.
```
